packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -485,6 +485,8 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin,
   _CompositeRenderEditablePainter _createBuiltInForegroundPainters() {
     return _CompositeRenderEditablePainter(
       painters: <RenderEditablePainter>[
+        _autocorrectHighlightPainter,
```

Comment:
Makes sense to me if this is what iOS does. However:
- if the user chooses to use an opaque color, it's going to cover the text
- the behavior is specific to iOS

There's a `paintCursorAboveText` parameter on RenderEditable that controls the paint order between the text and the cursor, maybe we can use that to control the paint order

Reply:
@LongCatIsLooong Hey, I change the painter order by the `paintCursorAboveText` parameter. And may be some golden pictures needs to change it bmn

Reply:
Could you fix the errors here: https://github.com/flutter/flutter/pull/96276/checks?check_run_id=5087904994 ?

Reply:
done